transcription/transcription.go

```diff
@@ -29,3 +30,15 @@ func msgHeaders(from string, to []string, subject string) string {
 	msgHeaders := []string{fromHeader, toHeader, subjectHeader}
 	return strings.Join(msgHeaders, "\r\n")
 }
+
+// ConvertAudioIntoRequiredFormat converts encoded audio into the required format.
+func ConvertAudioIntoRequiredFormat(fn string) error {
```

Comment:
Happening when I save file, even for old tests and even on `master`

![Uploading Screen Shot 2016-03-13 at 1.04.55 PM.png…]()